cmd/config.go

```diff
@@ -1,6 +1,8 @@
 package main
 
-import "github.com/xmidt-org/carousel"
+import (
```

Comment:
Since we're using the cmd directory structure, it'd be good to follow some of the standards that Go projects tend to have such as https://github.com/golang-standards/project-layout/tree/master/cmd

Some suggestions:
- Use path `cmd/carousel/cmd/[subCommandFolder]` or `cmd/carousel/[subcommand].go`
- Use `config` folder
- Use `internal` and `pkg` folders as appropiate.